### PR TITLE
refactor(gear-programs): upgrade sails version

### PR DESCRIPTION
**Pull request by ByteNacked**
- upgrade sails-rs version
- migrate tests

@reviewer-or-team

### CODE REVIEW

File: gear-programs/bridging-payment-vara-supply/client/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[package]
+name = "bridging-payment-vara-supply-client"
+version.workspace = true
+edition.workspace = true
+
+[dependencies]
+mockall = { version = "0.12", optional = true }
```
Comment (mertwole) [inline gear-programs/bridging-payment-vara-supply/client/Cargo.toml:7]: Workspace dependency

File: gear-programs/bridging-payment/client/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+[package]
+name = "bridging-payment-client"
+version.workspace = true
+edition.workspace = true
+
+[dependencies]
+mockall = { version = "0.12", optional = true }
```
Comment (mertwole) [inline gear-programs/bridging-payment/client/Cargo.toml:7]: Workspace deps

File: gear-programs/vft-client/Cargo.toml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+[package]
+name = "vft-client"
+version.workspace = true
+edition.workspace = true
+
+[dependencies]
+mockall = { version = "0.12", optional = true }
```
Comment (mertwole) [inline gear-programs/vft-client/Cargo.toml:7]: Workspace deps

File: gear-programs/vft-gateway/client/Cargo.toml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+[package]
+name = "vft-gateway-client"
+version.workspace = true
+edition.workspace = true
+
+[dependencies]
+mockall = { version = "0.12", optional = true }
```
Comment (mertwole) [inline gear-programs/vft-gateway/client/Cargo.toml:7]: Workspace deps

File: gear-programs/vft-treasury/client/Cargo.toml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+[package]
+name = "vft-treasury-client"
+version.workspace = true
+edition.workspace = true
+
+[dependencies]
+mockall = { version = "0.12", optional = true }
```
Comment (mertwole) [inline gear-programs/vft-treasury/client/Cargo.toml:7]: Workspace deps

File: gear-programs/bridging-payment-vara-supply/client/Cargo.toml (hunk 1 above)
Comment (ByteNacked) [reply gear-programs/bridging-payment-vara-supply/client/Cargo.toml]: Fixed

File: gear-programs/bridging-payment/client/Cargo.toml (hunk 2 above)
Comment (ByteNacked) [reply gear-programs/bridging-payment/client/Cargo.toml]: Fixed

File: gear-programs/vft-gateway/client/Cargo.toml (hunk 4 above)
Comment (ByteNacked) [reply gear-programs/vft-gateway/client/Cargo.toml]: Fixed

File: gear-programs/vft-treasury/client/Cargo.toml (hunk 5 above)
Comment (ByteNacked) [reply gear-programs/vft-treasury/client/Cargo.toml]: Fixed

File: gear-programs/vft-client/Cargo.toml (hunk 3 above)
Comment (ByteNacked) [reply gear-programs/vft-client/Cargo.toml]: Fixed

**Review by mertwole — APPROVED** [review]

**Pull request merged**